pkgs/development/tools/analysis/smatch/default.nix

```diff
@@ -21,6 +23,17 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-Pv3bd2cjnQKnhH7TrkYWfDEeaq6u/q/iK1ZErzn6bME=";
   };
 
+  patches = [
+    (let
+      clang-major = lib.versions.major (lib.getVersion llvmPackages.clang-unwrapped);
+      clang-lib = lib.getLib llvmPackages.clang-unwrapped;
+    in substituteAll {
+      src = ./fix_include_path.patch;
```

Comment:
Is this a custom patch by us or sourced from somewhere?

Reply:
It's a custom patch I wrote